src/libstd/io/mod.rs

```diff
@@ -2326,10 +2326,10 @@ impl<R: Read> Iterator for Bytes<R> {
 /// An iterator over the contents of an instance of `BufRead` split on a
 /// particular byte.
 ///
-/// This struct is generally created by calling [`split`][split] on a
-/// `BufRead`. Please see the documentation of `split()` for more details.
+/// This struct is generally created by calling [`split`] on a `BufRead`.
```

Comment:
While we're at it, can we link to BufRead here as well?

Reply:
I did not add `BufRead` because I thought it was intentional to reduce the numbers of links clickable. Of the links, `split` already directs to `BufRead` so people might just go back and click on this.

I think `split` should be preferred to be clicked over `BufRead`. In the meantime, I also noticed quite some links are not clickable even thought they are real types, I thought they are not linked purposely.

Should I still add it?